host/child_workflow_test.go

```diff
@@ -275,24 +267,31 @@ func (s *integrationSuite) TestCronChildWorkflowExecution() {
 		Identity:            identity,
 	}
 
+	// Because of rounding in GetBackoffForNextSchedule, we'll tend to stay aligned to whatever
```

Comment:
Can we fix the rounding issue instead? The rounding was needed because the backoff used to be seconds. Now I it is time.Duration so i believe rounding is no longer needed.

Reply:
I was going to try that later, but thought I should get the tests working without any behavior changes first. I can try to just remove the rounding if you think it's low risk. One small potential risk is that preserving the phase within the second is a very crude sort of jitter, that might be beneficial to users with huge numbers of cron workflows that start at the same time.